{"event": "verbose", "uuid": "55033d4d-947c-4c11-b0e9-f44687fa0596", "counter": 15, "stdout": "\u001b[0;31mThe error appears to be in '/root/demo/playbookone.yaml': line 11, column 1, but may\u001b[0m", "start_line": 14, "end_line": 15, "runner_ident": "d7ea8ad8-66f4-4a4e-8796-5321b6351329", "created": "2023-05-16T05:52:39.636238"}